external/kudu/pom.xml

```diff
@@ -27,5 +27,11 @@
       <version>${kudu.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
I can remove this, I added it when making an ephemeral test class for writing to Kudu.